C/common/storage_client.cpp

```diff
@@ -484,13 +484,26 @@ ReadingSet* StorageClient::queryTableToReadings(const std::string& tableName,
  * @return int		The number of rows inserted
  */
 int StorageClient::insertTable(const string& tableName, const InsertValues& values)
+{
+	return insertTable(DEFAULT_SCHEMA, tableName, values);
+}
+
+/**
+ * Insert data into an arbitrary table
+ *
+ * @param schema	The name of the schema to insert into
+ * @param tableName	The name of the table into which data will be added
+ * @param values	The values to insert into the table
+ * @return int		The number of rows inserted
+ */
+int StorageClient::insertTable(const string& schema, const string& tableName, const InsertValues& values)
 {
 	try {
 		ostringstream convert;
 
 		convert << values.toJSON();
 		char url[128];
-		snprintf(url, sizeof(url), "/storage/table/%s", tableName.c_str());
+		snprintf(url, sizeof(url), "/storage/schema/%s/table/%s", schema.c_str(), tableName.c_str());
```

Comment:
We can add a debug log here printing the "url" and message that insertion is done for particular schema and table, similar for other functions

Reply:
Not inclined to add this and we will end up swamping the logs